tools/wasi/CMakeLists.txt

```diff
@@ -1,14 +1,21 @@
 # Fizzy: A fast WebAssembly interpreter
-# Copyright 2019-2020 The Fizzy Authors.
+# Copyright 2020 The Fizzy Authors.
 # SPDX-License-Identifier: Apache-2.0
 
 include(ProjectUVWASI)
 
-set(fizzy_include_dir ${PROJECT_SOURCE_DIR}/lib/fizzy)
+add_library(wasi)
+add_library(fizzy::wasi ALIAS wasi)
+target_compile_features(wasi PUBLIC cxx_std_17)
```

Comment:
Not needed, `fizzy:fizzy` has it.

Reply:
The `fizzy::fizzy-internal` is private dependency: the `fizzy::wasi` will be built with `-std=c++17`, but users of `fizzy::wasi` (e.g. `fizzy-wasi` tool) will not have this requirement.

Here it make sense to declare `fizzy::wasi` with C++17 because it uses a C++17 feature in the main header.

Reply:
This is not relevant any more because `fizzy::wasi` now links `fizzy::fizzy-internal` publicly.

Reply:
> This is not relevant any more because `fizzy::wasi` now links `fizzy::fizzy-internal` publicly.

Why does it?

Reply:
Because now Fizzy is included in the `wasi.hpp`.